src/crate/theme/rtd/crate/sidebartoc.html

```diff
@@ -21,20 +21,22 @@
 
   {% if project == 'CrateDB: Tutorials' %}
     <li class="current">
-      <a class="current-active" href="{{ pathto(master_doc) }}">Self-Managed</a>
+      <a class="current-active" href="{{ pathto(master_doc) }}">Self-Managed (Legacy)</a>
       {{ toctree(maxdepth=3|toint, collapse=True, includehidden=theme_globaltoc_includehidden|tobool) }}
     </li>
   {% else %}
-    <li class="navleft-item"><a href="/docs/crate/tutorials/">Self-Managed</a></li>
+    <li class="navleft-item"><a href="/docs/crate/tutorials/">Self-Managed (Legacy)</a></li>
   {% endif %}
 
-  {% if project == 'CrateDB: How-Tos' %}
+  <li class="navleft-item"><a href="/docs/guide/install/">Self-Managed (NG)</a></li>
```

Comment:
Legacy vs. NG is just for demonstration purposes. Do not merge. We need to decide for the better one before merging.

Reply:
@ckurze: I clearly prefer [Self-Managed (NG)](https://cratedb.com/docs/guide/install/) over [Self-Managed (Legacy)](https://cratedb.com/docs/crate/tutorials/). Do you agree?

Reply:
Hi. We need your decision here. If you want to discuss internally, navigating to the link below is the right choice. Otherwise, I will always be happy if you follow my recommendations. If you have any better proposals, even later, we can always adjust, in order to improve further.

/cc @seut, @proddata, @ckurze

- https://github.com/crate/roadmap/issues/35#issuecomment-1782901011